app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str,
+                 ) -> None:
+        self.clean_mark = clean_mark
+        self.comfort_class = comfort_class
```

Comment:
In the requirements these attributes can accept only specific values. Let's check them and if value is incorrect, set some default correct value